src/components/Editor/helpers/setZoomEvent.ts

```diff
@@ -18,20 +18,37 @@ export const setZoomEvent = (
     const event = e as unknown as WheelEvent
     if (!(event.ctrlKey || event.metaKey)) return
 
-    event.preventDefault()
-
-    let fontSize = parseFloat(editorElement.dom.style.fontSize)
-    if (isNaN(fontSize)) {
-      fontSize = storedFontSize
-    }
+    changeFontSize(event, editorElement, storedFontSize, event.deltaY < 0)
+  })
 
-    if (event.deltaY < 0) {
-      fontSize = Math.min(fontSize + 1, maxFontSize)
-    } else {
-      fontSize = Math.max(fontSize - 1, minFontSize)
-    }
+  element.addEventListener('keypress', (e) => {
+    const event = e as unknown as KeyboardEvent
+    if (!(event.ctrlKey || event.metaKey)) return
+    if (event.key !== '+' && event.key !== '-') return
 
-    editorElement.dom.style.fontSize = `${fontSize}px`
-    localStorage.setItem('fontSize', `${fontSize}`)
+    changeFontSize(event, editorElement, storedFontSize, event.key === '+')
   })
 }
+
+function changeFontSize(
+  event: { preventDefault: () => void },
+  editorElement: EditorView,
+  storedFontSize: number,
+  increment: boolean
```

Comment:
Could you rename this variable to `isIncrement` or something that denotes a boolean value? Just to follow a specific convention!

Reply:
done :+1: